packages/astro/src/content/loaders.ts

```diff
@@ -16,6 +16,11 @@ export interface ParseDataOptions {
 	filePath?: string;
 }
 
+export type DataWithId = {
+	id: string;
+	[key: string]: unknown;
```

Comment:
Here, we define the data as `unknown`, but the zod schema is `z.any`. Is it intentional?

Reply:
I've tightened up the schema